docs/admin/config-guide/custom-scripts-config.md

```diff
@@ -363,3 +363,8 @@ For example, in our case; the command line is:
 ```
 
 That's all for `Custom Script` management with `CLI` feature.
+
+!!! Note
+
+If a custom script that is set as *Default authentication method* is disabled or deleted then the default authentication value will be removed.
```

Comment:
@pujavs if it is removed, what will be the fallback default authentication? Can AS live without any default authn method defined?

Reply:
@yuriyz, my understanding that AS can live without explicit setting of default auth method defined, request your confirmation and advice.

Reply:
Yes, if nothing works, and no scripts then AS will fallback to "simple_password_auth".